hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OMDBUpdatesHandler.java

```diff
@@ -153,8 +155,22 @@ private void processEvent(int cfIndex, byte[] keyBytes, byte[]
           }
         }
       } else if (action.equals(DELETE)) {
-        if (oldValue != null && !omUpdateEventValidator.isValidEvent(tableName,
-            oldValue, key, action)) {
+        if (null == oldValue) {
+          String keyStr = (key instanceof String) ? key.toString() : "";
+          if (keyStr.isEmpty()) {
+            LOG.warn(
+                "Only DTOKEN_TABLE table uses OzoneTokenIdentifier as key " +
+                    "instead of String. Event on any other table in this " +
+                    "condition may need to be investigated. This DELETE " +
+                    "event is on {} table which is not useful for Recon to " +
+                    "capture.", tableName);
+          }
+          LOG.warn("Old Value of Key: {} in table: {} should not be null " +
+              "for DELETE event ", keyStr, tableName);
+          return;
```

Comment:
@dombizita thanks for your review, Not sure If I understood correctly, but as I can see that if key is not empty, then also we are printing that "old value is null" information using message -> "Old Value of Key: {} in table: {} should not be null " , so I believe the purpose is still getting solved. Pls let me know if I understood something not correct.

Reply:
I wanted to say that with this solution I suggested we won't print it (as the `key instanceof String` will be false) and I'd like to know which approach is better: print it or not. I think it doesn't add any information with the empty key name, maybe we can adjust the second (in my suggestion) warn message to have something similar.

Reply:
@dombizita , I feel, current approach is better because if oldValue is null, which is more important for us, so we are printing the message that "Old value of key {} in table {} should not be null" and here we are printing both non-empty key as well as table name associated with that key which has its old value as null. This should be more helpful in debugging later. First log message which is getting printed is on condition `if (keyStr.isEmpty())` and this condition also is inside the condition` if (null == oldValue)` , so key can only be empty if it is not of string type and non string type is anyway not relevant for Recon to capture, but still this log message will be helpful and see if event is really based on `DTOKEN_TABLE` with what is intended for.

Reply:
Thanks for taking your time to answer here, I'm fine with this way!